lib/paperclip/interpolations.rb

```diff
@@ -30,7 +30,7 @@ def self.all
     def self.interpolate pattern, *args
       pattern = args.first.instance.send(pattern) if pattern.kind_of? Symbol
       all.reverse.inject(pattern) do |result, tag|
-        result.gsub(/:#{tag}/) do |match|
+        result.gsub(":#{tag}") do |match|
```

Comment:
Unused block argument - `match`. You can omit the argument if you don't care about it.